common/app/routes/challenges/redux/step-challenge-epic.test.js

```diff
@@ -56,7 +62,7 @@ test(file, function(t) {
           t.assert(
             onNextSpy.calledWithMatch({
               type: types.goToStep,
-              payload: 0
+              payload: { step: 0, isUnlocked: false }
```

Comment:
I don't know how to set this test up to return true.

Guidance would be greatly appreciated :+1: